app/GraphQL/Ecosystem/Types/Users.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\GraphQL\Ecosystem\Types;
+
+use Kanvas\Users\Models\Users as ModelsUsers;
+
+class Users
+{
+    /**
+     * Return the user address formatted for the response type.
+     *
+     * @param ModelsUsers $user
+     * @param array $request
+     * @return ModelsUsers
+     */
+    public function userAddress(ModelsUsers $user, array $request): ModelsUsers
```

Comment:
@arfenis since this is a user type class , lets just change the name to address and contact